local_scheme.py

```diff
@@ -0,0 +1,6 @@
+def local_scheme(version):
+    return (
```

Comment:
Could you use the old code for this function please? The changed version is harder to understand IMO.

Reply:
This code has been moved into setuptools-scm-node-and-dirty-tag, it could be moved back inline once https://github.com/pypa/setuptools_scm/issues/418 or https://github.com/pypa/setuptools_scm/issues/419 land